compiler/luci/pass/include/luci/Pass/RemoveNoEffectSlicePass.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_REMOVE_NO_EFFECT_SLICE_PASS_H__
+#define __LUCI_REMOVE_NO_EFFECT_SLICE_PASS_H__
+
+#include <logo/Pass.h>
+
+namespace luci
+{
+
+/**
+ * @brief  Class to Remove No Effect(input and output are same) Slice node.
+ */
+struct RemoveNoEffectSlicePass final : public logo::Pass
```

Comment:
How about using `RemoveRedundantSlicePass` ?

Reply:
um...what about `RemoveUnnecessarySlicePass`? Redundant have a meaning that have extra thing with same work.

Reply:
OK, `RemoveUnnecessarySlicePass` seems nice.

Reply:
Okay I will fix it Thanks!